ExternalTest/rr_sim_tests/rr_sim_tests/utils/wait_for_topics.py

```diff
@@ -0,0 +1,125 @@
+#! /usr/bin/env python3
+# Copyright 2020-2021 Rapyuta Robotics Co., Ltd.
```

Comment:
Do we need this file? Doesn't standard ROS2 has this?
https://github.com/ros2/launch_ros/pull/274/files

Reply:
Yes we do, that `wait_for_topics` was unfortunately only merged to master, not yet for our currently used [foxy](https://github.com/ros2/launch_ros/tree/foxy/launch_testing_ros/launch_testing_ros)

Reply:
Got it. could you write note somewhere?
Since it is Apatch2.0, putting here should be fine but better to be noted somewhere.

Reply:
Yeap, I did note [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/AddRRSimTests/ExternalTest/rr_sim_tests/rr_sim_tests/utils/wait_for_topics.py#L15), or you mean a note in README?

Reply:
Ah right, I did note [here](https://github.com/rapyuta-robotics/RapyutaSimulationPlugins/blob/AddRRSimTests/ExternalTest/rr_sim_tests/rr_sim_tests/utils/wait_for_topics.py#L15), or you mean a note in README?
Anyway, since it is Apache 2.0, actually I'm not sure how to further note, so could you help with it?

Reply:
I think current note is enough. thanks.